src/clientversion.h

```diff
@@ -19,7 +19,7 @@
 //! These need to be macros, as clientversion.cpp's and dynamic*-res.rc's voodoo requires it
 #define CLIENT_VERSION_MAJOR 2
 #define CLIENT_VERSION_MINOR 4
-#define CLIENT_VERSION_REVISION 0
+#define CLIENT_VERSION_REVISION 25
```

Comment:
0

Reply:
Will revert after merging this pull request using this commit: https://github.com/duality-solutions/Dynamic/commit/717f18d361993ecfc17415f4c8aaef2cd61c3793